web_warning_on_save/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of saving a record in the web interface.
+/!\/!\/!\ In no way this module stops the save of the record. You must consider this as a warning displayed 
+to the user AFTER save completed. /!\/!\/!\
```

Comment:
OK, that's the main reason of the module indeed. I only ask you to mention what I said.